portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro

```diff
@@ -19,6 +19,14 @@ definition {
 	macro addBreakdownByAttribute {
 		Click(locator1 = "ACSegments#ADD_BREAKDOWN_BUTTON");
```

Comment:
Hey @joshitagaki, I think you could move this line for after then `If` condition. What do you think?

Reply:
I wanted to keep it out of the if statement because there is a chance that another breakdown is present so it would require it to press the button

Reply:
Ohhh ok, thanks @joshitagaki